content/tracing/advanced_usage/_index.md

```diff
@@ -1232,11 +1236,11 @@ import datadog.trace.api.CorrelationIdentifier;
 
 // there must be spans started and active before this block.
 try {
-    ThreadContext.put("ddTraceID", "ddTraceID:" + String.valueOf(CorrelationIdentifier.getTraceId()));
-    ThreadContext.put("ddSpanID", "ddSpanID:" + String.valueOf(CorrelationIdentifier.getSpanId()));
+    ThreadContext.put("dd.trace_id", String.valueOf(CorrelationIdentifier.getTraceId()));
```

Comment:
Correct, the previous one was not right. `dd.trace_id` and `dd.span_id` are the right values we detect automatically.